ctapipe/image/hillas.py

```diff
@@ -690,7 +77,7 @@ def hillas_parameters_5(geom: CameraGeometry, image):
     width, length = np.sqrt(eig_vals)
 
     # psi is the angle of the eigenvector to length to the x-axis
-    psi = np.arctan2(eig_vecs[1, 1], eig_vecs[0, 1])
+    psi = np.arctan(eig_vecs[1, 1] / eig_vecs[0, 1])
```

Comment:
Ah I assume `test_with_toy()` (since it is a new test) discovered this little error and hence it was fixed within this PR?

Reply:
No, this was also discussed in Dortmund. It is basically the question what the support of the psi angle is. The size of the support was always π, but it was at different offsets.

Using np.arctan instead of arctan2 makes sure the interval is [-π/2, π/2]

Reply:
Not an error, just a very small improvement.

Reply:
Thanks for still writing it down here for reference